pkg/commands/compute/language_toolchain.go

```diff
@@ -179,6 +196,55 @@ func (bt BuildToolchain) Build() error {
 	return nil
 }
 
+// The encoding of a module starts with a preamble containing a 4-byte magic
+// number (the string '\0asm') and a version field.
+//
+// Reference:
+// https://webassembly.github.io/spec/core/binary/modules.html#binary-module
+func (bt BuildToolchain) validateWasm(path string) error {
+	// gosec flagged this:
+	// G304 (CWE-22): Potential file inclusion via variable
+	// Disabling as we trust the source of the path variable.
+	// #nosec
```

Comment:
Looks like gosec is happy with a constant because it can predictably identify the value as hard-coded.

Reply:
I've made the change but I personally prefer to inject dependencies wherever possible rather than rely on globals (although I appreciate a constant is a bit different in its intent)